### PR TITLE
[feat] add MiniCPM-V 4.6 backbone support

### PR DESCRIPTION
**Pull request by shaohua-pan**
## Summary

This PR adds OpenBMB MiniCPM-V 4.6 as a new VLM backbone for starVLA.

MiniCPM-V 4.6 is a lightweight multimodal model with a SigLIP2-400M vision encoder and Qwen3.5-0.8B language model, totaling about 1.3B parameters. This integration follows the existing VLM wrapper pattern used by Qwen-VL, Gemma4, and Molmo2, so existing VLM4A frameworks can use MiniCPM-V through the same `qwen_vl_interface` contract.

## Motivation

MiniCPM-V 4.6 provides a compact and efficient VLM option for VLA training and deployment. Compared with larger 4B/8B-class VLM backbones, it has lower memory and compute cost while still supporting image, multi-image, and video understanding.

This makes it useful as a lightweight VLM backbone baseline for robotics experiments and low-cost ablations.

## Changes

- Added `starVLA/model/modules/vlm/MiniCPM_V.py`
  - Implements `_MiniCPM_VL_Interface`
  - Loads MiniCPM-V 4.6 via `AutoModelForImageTextToText`
  - Supports `forward`, `generate`, and `build_qwenvl_inputs`
  - Exposes `model.config.hidden_size = text_config.hidden_size` for downstream action heads

- Updated `starVLA/model/modules/vlm/__init__.py`
  - Routes `minicpm-v` / `minicpmv` checkpoints to `_MiniCPM_VL_Interface`

- Added VLM4A framework wrappers
  - `starVLA/model/framework/VLM4A/MiniCPMPI.py`
  - `starVLA/model/framework/VLM4A/MiniCPMGR00T.py`

- Added MiniCPM examples
  - `examples/MiniCPM/README.md`
  - `examples/MiniCPM/run_libero_local_smoke.sh`
  - `examples/MiniCPM/submit_hpc3_libero.sh`
  - `examples/MiniCPM/eval_libero_local.py`

- Updated VLM module README
  - Added MiniCPM-V 4.6 to the supported VLM list

## LIBERO 4-Suite Benchmark Results

Config: MiniCPM-V 4.6 + PI head, 80K optimizer steps, effective BS=128, DeepSpeed ZeRO-2, `attn_implementation=sdpa`.

| Suite | Success Rate |
|---|---:|
| LIBERO-Spatial | 94.0% (470/500) |
| LIBERO-Object | 98.0% (490/500) |
| LIBERO-Goal | 98.0% (490/500) |
| LIBERO-10 | 92.4% (462/500) |
| **Average** | **95.6% (1912/2000)** |

Evaluation protocol: 50 trials per task, 10 tasks per suite, 2000 total rollouts.

## Training Setup

The submitted LIBERO training script follows the upstream starVLA training setup:

- Framework: `MiniCPMPI`
- Base VLM: `openbmb/MiniCPM-V-4.6`
- Data mix: `libero_all`
- Effective batch size: 128
  - `per_device_batch_size=16`
  - `num_processes=8`
  - DeepSpeed gradient accumulation = 1
- Max steps: 80000
- Attention implementation: `flash_attention_2 `
- DeepSpeed: ZeRO-2
- VLM unfrozen by default

## Test Plan

Syntax checks:

```bash
python3 -m py_compile \
  starVLA/model/modules/vlm/MiniCPM_V.py \
  starVLA/model/framework/VLM4A/MiniCPMPI.py \
  starVLA/model/framework/VLM4A/MiniCPMGR00T.py \
  examples/MiniCPM/eval_libero_local.py

bash -n examples/MiniCPM/run_libero_local_smoke.sh
bash -n examples/MiniCPM/submit_hpc3_libero.sh

### CODE REVIEW

**Review by JinhuiYE — APPROVED** [review]

**Comment by Junyu-Z** [comment]
Hi, I'm currently trying to reproduce your experiment results. Could you provide the final checkpoint?
